.github/ISSUE_TEMPLATE/release-blog-post.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Open Liberty release notes
+about: Checklist that must be completed to create and publish Open Liberty release notes.
+title: Release notes for VERSION_NUMBER
+labels: release
+assignees: lauracowen
+
+---
+
+This issue is to track the tasks involved in creating and publishing the releases notes for an Open Liberty 4-weekly release.
+
+The Open Liberty release notes are presented in three places:
+- [Open Liberty blog](https://openliberty.io/blog/) (GA and beta features; [example](https://openliberty.io/blog/2020/05/07/EJB-persistent-timers-20005.html))
+- [Red Hat Runtimes release notes for Open LIberty](https://access.redhat.com/documentation/en-us/open_liberty/2020/) (GA features only; [example](https://access.redhat.com/documentation/en-us/open_liberty/2020/html/release_notes_for_open_liberty_20.0.0.5_on_red_hat_openshift_container_platform/features))
+- [Red Hat Developer blog](https://developers.redhat.com/blog/) (GA features only; [example](https://developers.redhat.com/blog/2020/05/13/open-liberty-20-0-0-5-brings-updates-to-ejb-persistent-timers-coordination-and-failover-across-members/))
+
+
+## 1. Open Liberty release blog post
+
+The Open Liberty release blog post is written in asciidoc and pushed to the [OpenLiberty/blogs](https://github.com/openliberty/blogs) repo. The repo README contains [instructions on how to create, build, and review](https://github.com/OpenLiberty/blogs/blob/prod/README.md) the post, including a template file to help you.
+
+Follow the instructions in the README and the template file to complete the following tasks:
+
+- [ ] Create a draft release blog post in asciidoc, build to the draft site, and create a PR to `staging` with a link to the post on the draft site. Record all the reviews/approvals in the PR.
+- [ ] Get the draft post reviewed by:
+  - [ ] All the people who contributed the content to the blog post.
+  - [ ] At least one of the following people: @lauracowen, @mbroz2,@NottyCode
+- [ ] Agree with @lauracowen, @mbroz2, or @NottyCode which feature will lead this blog post, then write a title, slug, summary first paragraph, and SEO front matter appropriately.
+- [ ] Get the post approved by @lauracowen, @mbroz2, or @NottyCode.
+- [ ] On release day (usually a Friday), @lauracowen, @mbroz2, or @NottyCode will publish the post.
+
+## 2. Red Hat Runtimes release notes for Open Liberty
+
+The release notes are automatically built from the asciidoc release blog post. This is done by using the appropriate tags in the asciidoc source and then the Red Hat documentation build includes the tagged sections in the release notes.
+
```

Comment:
I would add a note here specifying that the main point of the tags is to remove references to the previews.

Reply:
I've added "The tags ensure that information in the blog post that is not relevant to the Red Hat Runtimes release are omitted from the Red Hat Runtimes release notes." though we might be able to simplify the tagging when we switch to separate beta posts. I'll update that later though.